### PR TITLE
WSTEAMA-371 Adds a unit test to Curation and Integration tests to check for the MostRead component

### PR DESCRIPTION
**Pull request by louisearchibald**
Resolves [JIRA 371](https://jira.dev.bbc.co.uk/browse/WSTEAMA-371)

Overall changes
======

- Adds a unit test to `src/app/components/Curation/getComponentName/index.test.tsx` to check that when a component has visualProminence of `NORMAL` and visualStyle of `RANKED` that we receive the `MOST_READ` component.
- Add integration tests for the homepage to check for the existence of the Most Read component. 

Code changes
======

- New unit test
- New integration tests

Testing
======

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/integration/pages/homePage/kyrgyz/__snapshots__/amp.test.js.snap
Comment (karinathomasbbc) [inline src/integration/pages/homePage/kyrgyz/__snapshots__/amp.test.js.snap]: Ah, this doesn't work on AMP... wonder if the test needs to be canonical only? (Because the AMP Most Read component needs to fetch the data, and interactivity doesn't work in Integration tests)

File: src/integration/pages/homePage/kyrgyz/__snapshots__/canonical.test.js.snap
Comment (karinathomasbbc) [inline src/integration/pages/homePage/kyrgyz/__snapshots__/canonical.test.js.snap]: Ideally we would not snapshot the entire component here - just the text content & URL... let's update the test

File: src/integration/pages/homePage/mostReadTests.js
Comment (karinathomasbbc) [inline src/integration/pages/homePage/mostReadTests.js]: I wonder if we could get this to check all the links?

I would like to do it in a similar way to https://github.com/bbc/simorgh/blob/28185d1173eeeff477dc363b6f0b7d95f056c17d/src/integration/common/appleItunesAppBanner.js

```suggestion
    it('has multiple items', () => {
      const mostReadLinks = mostRead.querySelectorAll('a');
      
      mostReadLinks.forEach(link, () => {
        const url = link.getAttribute('href');
        const text = link.textContent;
        
        expect({ url, text }).toMatchSnapshot();
      };
    });
```

File: src/integration/pages/homePage/kyrgyz/__snapshots__/canonical.test.js.snap
Comment (karinathomasbbc) [inline src/integration/pages/homePage/kyrgyz/__snapshots__/canonical.test.js.snap:736]: Nice!

File: src/integration/pages/homePage/pageContentTests.js
Comment (karinathomasbbc) [inline src/integration/pages/homePage/pageContentTests.js:2]: Thanks for fixing this (this was my fault)

**Review by karinathomasbbc — APPROVED** [review]
Let's not merge this until the migration PR has been merged (as it will add additional review overhead to an already massive PR 🙈 )

**Review by LilyL0u — APPROVED** [review]
I like it

**Review by pvaliani — APPROVED** [review]
Great work!

**Pull request merged**
